pkg/daemon/gpsd.go

```diff
@@ -22,7 +22,7 @@ const (
 	GNSSMONITOR_INTERVAL = 5 * time.Second
 )
 
-type gpsd struct {
+type GPSD struct {
```

Comment:
Is there a functional reason this change is necessary?

Reply:
i had to export the gpsd struct so pubsub framework can have access to it.
// Subscriber ... event subscriber
type Subscriber struct {
	source     event.EventSource
	gpsd       *GPSD
	monitoring bool
	id         string
}